src/audio/kpb.c

```diff
@@ -639,6 +639,8 @@ static int kpb_buffer_data(struct comp_dev *dev, struct comp_buffer *source,
 	if (kpb->state == KPB_STATE_DRAINING)
 		draining_data->buffered_while_draining += size_to_copy;
 
+	timeout = platform_timer_get(platform_timer) +
```

Comment:
Is it supposed to be 1 ms?

Reply:
Do you think it is too much? If buffering was interrupted that routine can take some time, so if we set timeout of 100us then I am afraid we will hit timeout frequently.

Reply:
I was afraid it can be too short, but if the tests show otherwise, then it's OK.